x-pack/plugins/index_management/public/application/sections/home/index_list/details_page/details_page_mappings_content.tsx

```diff
@@ -613,3 +622,78 @@ export const DetailsPageMappingsContent: FunctionComponent<{
     </>
   );
 };
+
+function hasSemanticTextField(fields: NormalizedFields): boolean {
+  return Object.values(fields.byId).some((field) => field.source.type === 'semantic_text');
+}
+
+function getStateWithCopyToFields(state: State): State {
```

Comment:
Should this moved to [utils.ts ](https://github.com/elastic/kibana/blob/main/x-pack/plugins/index_management/public/application/components/mappings_editor/lib/utils.ts) ? 
we could add some tests.